StreamChatClient_v3/WebSocketClient/Events/Event.swift

```diff
@@ -15,11 +26,16 @@ protocol EventWithPayload: Event {
 }
 
 /// A protocol for user `Event`s.
-protocol UserEvent: Event {
+protocol EventWithUserPayload: EventWithPayload {
     var userId: UserId { get }
 }
 
+/// A protocol for user owner `Event`s.
+protocol EventWithOwnerPayload: EventWithPayload {
+    var createdByUserId: UserId { get }
+}
+
```

Comment:
I don't understand what this is good for and the comment is not helping.

Reply:
This is the case where we have a user in an event and another user who generates it. For example, a user bans another user.

Reply:
What the benefit of having these kinds of protocols?

Reply:
I added comments.